src/_cffi_src/openssl/ssl.py

```diff
@@ -441,14 +432,8 @@
 #else
 static const long Cryptography_HAS_SECURE_RENEGOTIATION = 1;
 #endif
-#ifdef OPENSSL_NO_SSL2
+
 static const long Cryptography_HAS_SSL2 = 0;
```

Comment:
Could you put a comment that this exists for clients that may still check for SSLv2 support? We should probably figure out what a deprecation/removal policy is for things like this, but on the other hand the maintenance burden is essentially zero.